app/Mail/UnacceptedAssetReminderMail.php

```diff
@@ -29,11 +29,13 @@ public function __construct($checkout_info, $count)
      */
     public function envelope(): Envelope
     {
+        $this->count = $count;
         $from = new Address(config('mail.from.address'), config('mail.from.name'));
+        $subject = trans_choice('mail.unaccepted_asset_reminder', $count);
```

Comment:
I think `$this->count = $count;` should be removed and `$count` should be `$this->count` on line 34.